doc/api/screen_handling.rst

```diff
@@ -0,0 +1,83 @@
+.. _screen_handling_label:
+
+Screen Handling
+===============
+
+.. automodule:: simpleline.render.screen_handler
+
+The :class:`ScreenHandler` class is used to schedule
+:class:`UIScreen <simpleline.render.screen.UIScreen>` to the screen stack.
+
+Screen handling is an important part of using Simpleline library and it is recommended for user
+to get familiar with this principle. Screen handler is gateway for a developer to manage stack so
+screens can be placed to
+this stack and removed from this stack. There are multiple ways how to place screen to the stack,
+these will be described below. To remove screen from the stack the screen must be closed or it can
+be replaced by other screen. To close :class:`UIScreen <simpleline.render.screen.UIScreen>`
+a developer could call the :meth:`UIScreen.close() <simpleline.render.screen.UIScreen.close()>`
+method or a screen can be closed by a user by pressing `c` (can be disabled) to continue which will
+close the screen automatically. When a screen is closed then next top screen on the stack will be
```

Comment:
`closed then next top screen on the stack` -> `closed, the next screen on the top of the stack`